cgi-bin/DW/Controller/Talk.pm

```diff
@@ -864,9 +865,15 @@ sub preview_parent_args {
 
         my $poster      = 'Anonymous';
         my $poster_name = '';
+        my $in_journal  = $entry->journal->ljuser_display;
+
         if ( $parentitem->poster ) {
             $poster      = $parentitem->poster->ljuser_display;
             $poster_name = $parentitem->poster->name_html;
+
+            if ( $parentitem->poster->user eq $entry->journal->user ) {
```

Comment:
nit: I think it's preferable to say `$parentitem->poster->equals($entry->journal)` here.